packages/odie-client/src/data/use-odie-chat.ts

```diff
@@ -3,16 +3,22 @@ import apiFetch from '@wordpress/api-fetch';
 import wpcomRequest, { canAccessWpcomApis } from 'wpcom-proxy-request';
 import { useOdieAssistantContext } from '../context';
 import { generateUUID } from '../utils';
+import { useCurrentSupportInteraction } from './use-current-support-interaction';
 import type { OdieChat, ReturnedChat } from '../types';
 
 /**
  * Get the ODIE chat and manage the cache to save on API calls.
  */
 export const useOdieChat = ( chatId: number | null ) => {
-	const { botNameSlug, version } = useOdieAssistantContext();
+	const { version } = useOdieAssistantContext();
+	const { data: supportInteraction } = useCurrentSupportInteraction();
+
+	// Fallback to `wpcom-support-chat` in case this is an old interaction without bot_slug property.
+	// In the Help Center, up to October 2025, all interactions were created with `wpcom-support-chat` bot.
+	const botSlug = supportInteraction?.bot_slug || 'wpcom-support-chat';
```

Comment:
Wdyt about getting `wpcom-support-chat` as a const from the defaultContext so we domn't have to repeat it?